pkg/sloop/server/internal/config/config_test.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"testing"
+	"encoding/json"
+	"github.com/ghodss/yaml"
+)
+
+func Test_loadFromJSONFile(t *testing.T) {
+	var expectedconfig SloopConfig
+	configfilename:="testconfig.json"
+	configFile, err := ioutil.ReadFile(configfilename)
+	err = json.Unmarshal(configFile, &expectedconfig)
+	assert.Nil(t,err)
+
+	out_config,err := loadFromFile(configfilename)
+
+	assert.Nil(t,err)
+	assert.Equal(t,out_config,&expectedconfig)
+}
+
+func Test_loadFromYAMLFile(t *testing.T) {
+	var expectedconfig SloopConfig
+	configfilename:="testconfig.yaml"
+	configFile, err := ioutil.ReadFile(configfilename)
+	err = yaml.Unmarshal(configFile, &expectedconfig)
+	assert.Nil(t,err)
+
+	out_config,err := loadFromFile(configfilename)
+
+	assert.Nil(t,err)
+	assert.Equal(t,out_config,&expectedconfig)
+}
+
+func Test_loadFromTxtFile(t *testing.T) {
```

Comment:
please use the following naming for test functions: `Test_<action>_<expectedResult>`
for example: `Test_loadFromTextFile_Failure`, `Test_loadFromJson_Success`.